packages/components/src/DateTimePickers/Time/Manager/Manager.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TimeContext } from '../Context';
+import extractTime from '../time-extraction';
+
+class ContextualManager extends React.Component {
+	static displayName = 'Time.Manager';
+	static propTypes = {
+		children: PropTypes.node,
+		onChange: PropTypes.func,
+		useSeconds: PropTypes.bool,
+		selectedTime: PropTypes.string,
```

Comment:
Can you name it value please ? It will fit more the fact that the main element is an input. The picker is just a tool to set the input `value`